saleor/checkout/utils.py

```diff
@@ -197,13 +197,15 @@ def add_variants_to_checkout(
     to_reserve = to_create + to_update
     if reservation_length and to_reserve:
         updated_lines_ids = [line.pk for line in to_reserve + to_delete]
+        to_update_reserved_until = []
+
         for line in checkout_lines:
             if line.pk not in updated_lines_ids:
-                to_reserve.append(line)
-                variants.append(line.variant)
+                to_update_reserved_until.append(line)
```

Comment:
I will suggest adding a comment here, is hard to understand what is it for without context.